src/onboarding/components/CompletionStep.tsx

```diff
@@ -116,8 +116,8 @@ class CompletionStep extends PureComponent<Props> {
                         />
                         <dt>I've got this...</dt>
                         <dd>
-                          Jump into InfluxDB Cloud and set up data collection
-                          when you’re ready.
+                          Jump into InfluxDB and set up data collection when
+                          you’re ready.
```

Comment:
why are these on two separate lines? Also, the previous version had `when` on the second line?

Reply:
Prettier changed this to move "when" up because there were too few characters on line 119 after "Cloud" was removed.  It wasn't a conscious decision on my part.  You can see that the change was introduced in my commit "run prettier" - all I did on that commit was run `yarn prettier:fix`.